.travis.yml

```diff
@@ -3,25 +3,19 @@ addons:
     packages:
     - nodejs
 language: python
+python:
+  - 3.5
+  - 3.6
```

Comment:
Shouldn't need the 3.6 tests.

Reply:
3.6 tests were already there in the Travis so I kept them. Should I remove 3.6?

Reply:
It's not urgent to change, but it's wasting Travis resources testing a Python version we probably won't use (we currently plan to skip straight to 3.8).

Reply:
Removed python3.6 tests to save Travis resources as per suggestion.